editor/editor_export.cpp

```diff
@@ -1408,6 +1408,37 @@ bool EditorExport::poll_export_platforms() {
 	return changed;
 }
 
+void EditorExport::add_export_console_placeholder_platforms() {
+	{
+		Ref<EditorExportConsolePlaceholder> ps;
+		ps.instance();
+		ps->console_name = "Sony PlayStation";
```

Comment:
Should this be PlayStation or PlayStation 4? I'm not sure if we want to add a placeholder for each iteration of the console, especially as the PlayStation 5 will be released soon.

Reply:
It should be PlayStation 4, Xbox One and Nintendo Switch, as this is the current-gen.
Next-gen platform support placeholders should be added when those will be officially supported by anyone.

That being said, none of the listed Godot porting third-party companies currently provide support for PS4 so for transparency sake I don't think this platform should be mentioned anywhere at this time.

Reply:
With 5 coming out, it makes no sense to put a number here IMO.